ibis/expr/types/logical.py

```diff
@@ -48,21 +48,184 @@ def ifelse(
+        >>> t = ibis.memtable({"arr": [1, 2, 3, None]})
+        >>> (t.arr > 1).__or__(t.arr > 2)
+        ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+        ┃ Or(Greater(arr, 1), Greater(arr, 2)) ┃
+        ┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+        │ boolean                              │
+        ├──────────────────────────────────────┤
+        │ False                                │
+        │ True                                 │
+        │ True                                 │
+        │ NULL                                 │
+        └──────────────────────────────────────┘
+        """
         return _binop(ops.Or, self, other)
 
     __ror__ = __or__
 
     def __xor__(self, other: BooleanValue) -> BooleanValue:
+        """Construct a binary XOR conditional expression with `self` and `other`.
+
+
+        Parameters
+        ----------
+        self
+            Left operand
+        other
+            Right operand
+
+        Returns
+        -------
+        BooleanValue
+            A Boolean expression
+
+        Examples
+        --------
+        >>> import ibis
+        >>> ibis.options.interactive = True
+        >>> t = ibis.memtable({"arr": [1, 2, 3, None]})
+        >>> (t.arr == 2)
+        ┏━━━━━━━━━━━━━━━━┓
+        ┃ Equals(arr, 2) ┃
+        ┡━━━━━━━━━━━━━━━━┩
+        │ boolean        │
+        ├────────────────┤
+        │ False          │
+        │ True           │
+        │ False          │
+        │ NULL           │
+        └────────────────┘
+
+        >>> (t.arr > 2)
+        ┏━━━━━━━━━━━━━━━━━┓
+        ┃ Greater(arr, 2) ┃
+        ┡━━━━━━━━━━━━━━━━━┩
+        │ boolean         │
+        ├─────────────────┤
+        │ False           │
+        │ False           │
+        │ True            │
+        │ NULL            │
+        └─────────────────┘
+
+        >>> (t.arr == 2).__xor__(t.arr > 2)
+        ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+        ┃ Xor(Equals(arr, 2), Greater(arr, 2)) ┃
+        ┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+        │ boolean                              │
+        ├──────────────────────────────────────┤
+        │ False                                │
+        │ True                                 │
+        │ True                                 │
+        │ NULL                                 │
+        └──────────────────────────────────────┘
+        """
+
         return _binop(ops.Xor, self, other)
 
     __rxor__ = __xor__
 
     def __invert__(self) -> BooleanValue:
+        """Construct a unary NOT conditional expression with `self`.
+
+
+        Parameters
+        ----------
+        self
+            Operand
+
+        Returns
+        -------
+        BooleanValue
+            A Boolean expression
+
+        Examples
+        --------
+        >>> import ibis
+        >>> ibis.options.interactive = True
+        >>> t = ibis.memtable({"arr": [True, False, False, None]})
+        >>> t.arr.__invert__()
```

Comment:
```suggestion
        >>> ~t.arr
```

Reply:
@anjakefala Looks like maybe this didn't get picked up in the changes you pushed?

Reply:
Good eye!